src/ontology/uberon-edit.obo

```diff
@@ -224035,6 +224035,16 @@ is_a: UBERON:0009569 ! subdivision of trunk
 relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date "2023-05-16T06:47:28Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480062
+name: femoral head cartilage
+def: "Hyaline articular cartilage covering the head of femur." [PMID:36292459]
+is_a: UBERON:0010996 ! articular cartilage of joint
```

Comment:
Would this term also be a SubclassOf [hyaline cartilage tissue](http://purl.obolibrary.org/obo/UBERON_0001994)?

Reply:
@aleixpuigb articular cartilage of joint is_a hyaline cartilage tissue, that's why I did not add it as SubClass. Is that right?

Reply:
You are correct, sorry for missing it.